avatar/avatar.css

```diff
@@ -0,0 +1,16 @@
+
+.avatar{
+    margin: 0.5rem;
+}
+.avatar img{
+    border-radius: 50%;
+}
+.sm{
+    width: 2rem;
+}
+.md{
+    width: 3rem;
+}
```

Comment:
you can give classname as `avatar-sm` instead of `sm` if you are using this class specifically for avatar

Reply:
sure I will do that in Final refactor code base